docs/contributor_guide/development_process.rst

```diff
@@ -7,6 +9,18 @@ Contributors should use their own forks of the repository. In their fork, they
 create feature branches off of :code:`master`, and their pull requests should
 target the :code:`master` branch. Maintainers are responsible for prompt
 review of pull requests.
+The `CODEOWNERS.md <https://github.com/fairlearn/fairlearn/blob/master/CODEOWNERS.md>`_
+file lists the subject matter experts for all parts of the repository
+including the documentation and examples.
+Every change requires approval from at least one of the listed subject matter
+experts, and a total of two approvals from current maintainers.
+All changes to APIs should be reviewed by
+`Miro Dudik <https://github.com/MiroDudik>`_.
```

Comment:
I don't think we need to mention the codeowners file if that's just for the auto suggestion of reviews. And I'm not sure why we're singling out Miro here.

Reply:
As it states there we should have one person from the codeowners list approve, e.g., if Michael M. and I approve a PR on metrics that should probably get another look from someone else.

This part of the PR was written a while ago and we can certainly change this. For example, if you don't feel like we should treat the codeowners file as a list of subject matter experts we can adjust this text. At the point when I wrote this I asked everyone what PRs they'd want to be reviewers on, and CODEOWNERS.md is a reflection of that. That's why it explicitly says that Miro wants to be a reviewer on API changes.

Reply:
(Removed the API review special case here and in codeowners file)
Does the "at least one subject matter expert" rule make sense? Perhaps it's too obvious not to complete PRs before someone with expertise in that code has reviewed it, but I don't think it hurts to have that written down.